public/js/admin.js

```diff
@@ -25,12 +25,25 @@ async function getNotes() {
     document.querySelector('#notes').innerHTML = html;
 }
 
+// eslint-disable-next-line
```

Comment:
Why did this need to be added?

Reply:
Because eslint can't see that these functions are actually called because they're not called here. It's on every function

Reply:
The correct way to do it is like this:

https://github.com/rpiambulance/headsup/blob/babee5aeed4a9f24f0d3b1470d728c68accc07df/public/js/whiteboard.js#L1

Reply:
(at the top of the file)

Reply:
This is if the function is defined outside the file how to make ES lint happy about us using it inside the file and not saying it is undefined. This doesn't work the other way which is what we need here. We need to say hey we define functions in this file, but I promise you they're being used somewhere

Reply:
Will this work?

https://eslint.org/docs/rules/no-unused-vars#exported